spec/dummy/app/controllers/api/v1x0.rb

```diff
@@ -21,5 +21,22 @@ def show
         render :json => {:id => request_path_parts["primary_collection_id"]}.to_json
       end
     end
+
+    class PersonsController < ApplicationController
+      def create
+        params_for_create
```

Comment:
What does this test besides just running the methods?

Reply:
@Fryguy It just calls the strong parameters check and if its bad we get a 400 back.

Reply:
Still feels like there should be a render call or something...Standalone it doesn't feel like a realistic looking tests.

Reply:
@Fryguy Please review I have added a check for openapi.